src/components/Player/ClueText.js

```diff
@@ -42,3 +42,8 @@ export default ({text = ''}) => {
     </>
   );
 };
+
+test('decodeHtml', () => {
```

Comment:
Looks good! it works well for Tres but for vis-a-vis i think i get an error when i try it in the javascript console. Maybe add a try catch, and fallback to just returning text.value?

Though test is not defined: https://downforacross-dhcne081x-downforacross1.vercel.app/beta/game/3546695-ust

<img width="332" alt="image" src="https://github.com/downforacross/downforacross.com/assets/1988030/005db6bc-d38b-4654-af02-8f370adddc7c">


I'm ok if you just delete these lines... or comment it out

Reply:
@sweep-ai ptal